MobileApplication/MobileApplication/Views/AboutPage.xaml.cs

```diff
@@ -19,8 +19,11 @@ namespace MobileApplication.Views
 
     public partial class AboutPage : ContentPage
     {
-        private NewCategoryPage _newCategory;
         public CategoryPage categoryPageNew;
+        private float net;
```

Comment:
What is `net`, what it stands for? And why it's a float, isn't it some monetary amount?

Reply:
'net' stands for "Net income". It's float because microcharts only support float variables to represent data

Reply:
So if micro charts support only float `net` should be converted to float only  **after** all calculations (where you use entries add)

Reply:
okay, i will set net for decimal and only make it float with entries 👍